docs/src/test/scala/docs/http/scaladsl/HttpClientExampleSpec.scala

```diff
@@ -148,10 +148,10 @@ class HttpClientExampleSpec extends WordSpec with Matchers with CompileOnlySpec
         val responseFuture: Future[HttpResponse] = dispatchRequest(HttpRequest(uri = "/"))
 
         responseFuture.andThen {
-          case Success(_) => println("request succeeded")
-          case Failure(_) => println("request failed")
+          case Success(_) ⇒ println("request succeeded")
```

Comment:
Hm... In Akka we usually do not want to show the fancy arrows in documentation; could we fix this before merging?